services/app-api/utils/testing/fixtures/completionStatus/mcpar-complete.result.test.json

```diff
@@ -35,7 +35,8 @@
     },
     "/mcpar/plan-level-indicators/quality-measures": true,
     "/mcpar/plan-level-indicators/sanctions": true,
-    "/mcpar/plan-level-indicators/program-integrity": true
+    "/mcpar/plan-level-indicators/program-integrity": true,
+    "/mcpar/plan-level-indicators/ilos": true
```

Comment:
do we need `/mcpar/program-information/add-in-lieu-of-services-and-settings` in program-information as well?

Reply:
good question! we don't need this for that route because the entity of that page is actually "ILOS" which is not required 😄